packages/dom/src/clone-dom.js

```diff
@@ -42,20 +42,19 @@ const deepClone = (host, disableShadowDOM) => {
     walkTree(node.firstChild, clone);
   };
 
-  let fragment = document.createDocumentFragment();
-  cloneNode(host, fragment);
+  let fragment = dom.createDocumentFragment();
+  cloneNode(dom.documentElement, fragment);
+  fragment.documentElement = fragment.firstChild;
+  fragment.head = fragment.querySelector('head');
+  fragment.body = fragment.querySelector('body');
   return fragment;
 };
 
 /**
  * Deep clone a document while also preserving shadow roots and converting adoptedStylesheets to <style> tags.
  */
 const cloneNodeAndShadow = (ctx) => {
-  let cloneDocumentFragment = deepClone(ctx.dom.documentElement, ctx.disableShadowDOM);
-  cloneDocumentFragment.documentElement = cloneDocumentFragment.firstChild;
-  cloneDocumentFragment.head = cloneDocumentFragment.querySelector('head');
-  cloneDocumentFragment.body = cloneDocumentFragment.querySelector('body');
-  return cloneDocumentFragment;
+  return deepClone(ctx.dom, ctx.disableShadowDOM);
 };
```

Comment:
This function seems redundant, we can simply do `deepClone(ctx)`

Reply:
addressed https://github.com/percy/cli/pull/1192/commits/228a6c6cf15c799daa7cd5931bb3b23dc0a0c2d7